lib/domain/db/cobble_database.dart

```diff
@@ -6,13 +6,13 @@ import 'dao/timeline_pin_dao.dart';
 
 void createAllCobbleTables(Database db) async {
   await db.execute("""
-    CREATE TABLE $TABLE_TIMELINE_PINS(
+    CREATE TABLE $tableTimelinePins(
```

Comment:
I don't understand how this works but is sql injection possible here?

Reply:
No, `tableTimelinePins` is constant.